pure_pursuit_wrapper/launch/pure_pursuit_wrapper.launch

```diff
@@ -9,6 +9,7 @@
         <include file="$(find pure_pursuit)/launch/pure_pursuit.launch">
             <arg name="is_linear_interpolation" value="True"/>
             <arg name="publishes_for_steering_robot" value="True"/>
+            <rosparam command="load" file="$(arg vehicle_calibration_dir)/pure_pursuit/calibration.yaml"/>
```

Comment:
Does this run?
rosparam is not part of the supported xml set for the include tag
http://wiki.ros.org/roslaunch/XML/include

Reply:
Yeah, that makes sense. It's supported under the group tag. I'll move it back outside

Reply:
If you put in above the desired scope it will not be loaded into the namespace you want. Since all the these parameters are private they need to end up under the namespace of the pure_pursuit node.